readme.md

```diff
@@ -577,7 +577,7 @@
 - [Visual Regression Testing](https://github.com/mojoaxel/awesome-regression-testing) - Ensures changes did not break the functionality or style.
 - [Theravada](https://github.com/johnjago/awesome-theravada) - Teachings from the Theravada Buddhist tradition.
 - [inspectIT](https://github.com/inspectit-labs/awesome-inspectit) - Open source Java app performance management tool.
-
+- [Numerical software](https://github.com/nschloe/awesome-numerical-software) - Software for numerical analysis.
```

Comment:
I think you should describe, briefly, what numerical analysis is.

Reply:
I'm not sure if this is helpful. The dictionary says

> [Numerical analysis is] the branch of mathematics that deals with the development and use of numerical methods for solving problems.

which doesn't help anyone who doesn't know what _numerical_ means, and Wikipedia says

> Numerical analysis is the study of algorithms that use numerical approximation (as opposed to general symbolic manipulations) for the problems of mathematical analysis (as distinguished from discrete mathematics).

which is way to verbose for awesome.

Perhaps you have suggestion?

Reply:
I have thought about this as well: I don't see a brief (I.e. one sentence) explanation of numerical analysis that helps laypeople understand what the field is about. And for people in the know, it could come off as tautology.

Reply:
Random thought. How about "Mathematical analysis using numerical approximations" as the description?